lib/rzo/app/templates/environment.conf.erb

```diff
@@ -0,0 +1,18 @@
+# Each environment can have an environment.conf file. Its settings will only
```

Comment:
Just as an FYI too, the bootstrap script installs the file using:

```shell
install -o root -g puppet -m 0644 ${BASE_DIR}/environment.conf ${PUPPET_ENVS}/${ENVIRONMENT}/environment.conf
```

This evaluates to:

```
install -o root -g puppet -m 0644 \
  /tmp/bootstrap_puppet4/environment.conf \
  /etc/puppetlabs/code/environments/production/environment.conf
```